tests/unit/amo/sagas/test_guides.js

```diff
@@ -0,0 +1,93 @@
+import SagaTester from 'redux-saga-tester';
+
+import { fetchGuidesAddons } from 'amo/reducers/guides';
+import * as searchApi from 'core/api/search';
+import addonsReducer, { loadAddonResults } from 'core/reducers/addons';
+import guidesSaga from 'amo/sagas/guides';
+import apiReducer from 'core/reducers/api';
+import {
+  dispatchClientMetadata,
+  createStubErrorHandler,
+  fakeAddon,
+} from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  const guid = 'support@lastpass.com,{b76ed4e7-12a6-4f25-a27b-fc3f93289008}';
+
+  let errorHandler;
+  let mockApi;
+  let sagaTester;
+
+  beforeEach(() => {
+    errorHandler = createStubErrorHandler();
+    mockApi = sinon.mock(searchApi);
+    const initialState = dispatchClientMetadata().state;
```

Comment:
Can you have a blank line before and after that line?

Reply:
maybe to help avoid this in the future (since I see this a lot it seems), when should I have blank lines?